src/components/Progress/Progress.stories.js

```diff
@@ -11,5 +11,29 @@ const Wrapper = styled.div`
 `;
 storiesOf('Progress', module)
   .addDecorator(story => <Wrapper>{story()}</Wrapper>)
-  .add('default', () => <Progress percent={52} />)
-  .add('no shadow', () => <Progress percent={52} shadow={false} />);
+  .add('default', () => <ProgressExample />)
+  .add('indeterminate', () => <Progress variant='indeterminate' />)
+  .add('hide value', () => <Progress hideValue value={34} />);
+
+const ProgressExample = () => {
+  const [percent, setPercent] = useState(0);
+
+  React.useEffect(() => {
+    function progress() {
```

Comment:
Since `progress` is only being used inside the `setInterval`, what do you think about declaring it as an anonymous function inside of `setInterval` directly?

Reply:
fixed in 2d306d1